pkg/metrics/record.go

```diff
@@ -27,6 +27,9 @@ import (
 	"kpt.dev/configsync/pkg/status"
 )
 
+// Initialization is the tag value to initiate a Count
+const Initialization = "cs_counter_initialization"
```

Comment:
Seems weird to just have one of these. might make sense as an enum of operations? Or maybe we should just remove the dead arg, since we don't send it any more.

Reply:
Shouldn't RecordResourceFight send 0 instead of 1 when initializing?

Reply:
This part has been moved to https://github.com/GoogleContainerTools/kpt-config-sync/pull/1237, will be making change accordingly.